docs/_includes/attr-builtin.adoc

```diff
@@ -532,21 +532,19 @@ Can also be set implicity from the revision line.
 |{y}
 |<<docbook>>
 
+|outfilesuffix
+|File extension to add to output files. (including leading period).
+|_empty_
+|_empty_
+|
+|<<navigating-between-source-files>>
```

Comment:
I like adding outfilesuffix to the built-in attributes table since it is an attribute that can be controlled. However, I don't think we should remove it from the intrinsic attribute table since it does get a default value based on converter being used. It's one of those attributes that lives in both world. Could you restore the entry in attr-data.adoc?